Components/Trips/Trips.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Text, View, FlatList } from 'react-native';
+import TripPreview from '../TripPreview/TripPreview';
+
+const mockPreviews = [
+  {
+    id: 1,
+    name: 'spring break',
+    startDate: '03-05-2020',
+    endDate: '04-05-2020',
+    origin: 'denver, co, usa',
+    finalDestination: 'paris, france',
+    originAbbrev: 'DEN',
+    finalDestinationAbbrev: 'PAR'
+  },
+  {
+    id: 2,
+    name: 'Summer Roadtrip',
+    startDate: '06-05-2020',
+    endDate: '07-05-2020',
+    origin: 'denver, co, usa',
+    finalDestination: 'Barcelona, spain',
+    originAbbrev: 'DEN',
+    finalDestinationAbbrev: 'BRC'
+  },
+  {
+    id: 3,
+    name: 'Winter Trip',
+    startDate: '11-05-2020',
+    endDate: '12-05-2020',
+    origin: 'denver, co, usa',
+    finalDestination: 'Tokyo, Japan',
+    originAbbrev: 'DEN',
+    finalDestinationAbbrev: 'TOK'
+  }
+];
+
+const Trips = () => {
+  return (
+    <View>
+      <Text>hey</Text>
+      <FlatList 
+        data={mockPreviews}
+        renderItem={({item}) => (<TripPreview item={item} key={item.id} />)}
```

Comment:
Still wondering why data needs to come in as an item property, but I am glad we got it working.